### PR TITLE
Fix for excluding mndt format

### PR DESCRIPTION
**Pull request by SiMet**
Remove format from mndt.

### CODE REVIEW

**Comment by stephanesan** [comment]
Because this is not a recognized format, we should move the definition of this oic type (i.e. it's pattern) in  https://github.com/openconnectivityfoundation/core/blob/master/schemas/oic.types-schema.json to foster re-use of the definition.

**Comment by SiMet** [comment]
The main problem is that format says it should be date-time. However, pattern is for date

**Comment by stephanesan** [comment]
Yet the fix should follow the design pattern of the project. If there doesn't exist a predefined Json schema format definition (as in your path), then an oic type is defined and put in the oic.type-schema.json file referenced below for reuse by other schemas.
My comment did not mean to question your fix but just asking to (at the same time) follow the design pattern of the project and move the pattern under its own definition (named date) in the oic type file.

-------- Original message --------

File: schemas/oic.wk.p-schema.json [hunk 1]
```diff
@@ -34,7 +34,6 @@
           "readOnly": true,
           "description": "ReadOnly, Manufacturing Date as defined in ISO 8601, where the format is [yyyy]-[mm]-[dd].",
           "pattern": "^([0-9]{4})-(1[0-2]|0[1-9])-(3[0-1]|2[0-9]|1[0-9]|0[1-9])$",
```
Comment (stephanesan) [inline schemas/oic.wk.p-schema.json]: trailing comma should be removed

**Comment by stephanesan** [comment]
I've implemented the suggestion above and pulled in these two fixes into a new pull request https://github.com/openconnectivityfoundation/core/pull/15. So I am closing this one now.

**Pull request closed without merging**